webapp/src/client/client.ts

```diff
@@ -2,11 +2,16 @@
 // See LICENSE.txt for license information.
 
 import HttpClient from 'client/httpClient';
+import {types} from 'sass';
```

Comment:
This is the right place where to import the error type? Sounds weird.

Reply:
It was unused import, removed.